unfurl/job.py

```diff
@@ -487,7 +487,7 @@ def run(self):
                     continue
 
                 if self.jobOptions.planOnly:
-                    errors = self.cantRunTask(task)
+                    errors = self.canRunTask(task)
```

Comment:
@jozo this what I mean: now canRunTask() return false to indicate it can run... that is backwards.

Reply:
You're right. I changed it to "go style" - now the method returns both True/False and errors. It should make more sense.